README.md

```diff
@@ -2,5 +2,7 @@
 
 A model of the Solar system using PyGameZero. Made for fun and fiddling.
 
+![Orrery Screenshot](https://github.com/user-attachments/assets/291c7777-836f-479a-8266-a02a957236d1)
```

Comment:
I'm not sure it's right to just reference the user-attachment here, I could be wrong, but I don't know that this is a permanent link. 
Can a copy of that image be put in a new screenshots folder and referenced from there? That also keeps the Git repo self contained.
Thanks